clients/js/src/cmds/info/rpc.ts

```diff
@@ -18,14 +13,12 @@ export const builder = (y: typeof yargs) =>
     } as const)
     .positional("chain", {
       describe: "Chain to query",
-      choices: Object.keys(CHAINS) as ChainName[],
+      type: "string",
```

Comment:
chain choices 🙏

Reply:
I feel like @barnjamin and @kcsongor should come to an agreement on this and I will implement that.

Reply:
Changed